chaoslib/litmus/platform/gke/node_cpu_consumption.yml

```diff
@@ -25,61 +35,42 @@
         message: "Injecting {{ c_experiment }} chaos on {{ node_name }} node"
       when: "c_engine is defined and c_engine != ''"
       
-    ## CREATING  A DAEMONSET  FOR CPU-HOG
-    - name: Creating a DaemonSet for cpu-hog
+    - name: Wait for the specified ramp time before injecting chaos
+      wait_for: timeout="{{ ramp_time }}"
+      when: "ramp_time is defined and ramp_time != ''"
+
+    ## CREATING A JOB FOR CPU-HOG
+    - name: Creating a Job for cpu-hog
       k8s:
         state: present
         namespace: "{{ a_ns }}"
-        src: /chaoslib/litmus/platform/gke/node-cpu-hog-daemonset.yml
-      register: cpu_hog_ds_result
+        src: /chaoslib/litmus/platform/gke/node-cpu-hog-job.yml
+      register: node_cpu_hog_job_result
```

Comment:
After creation of job can you add a task which will till the completion of job.  Refer https://github.com/litmuschaos/litmus/blob/master/chaoslib/pumba/network_chaos/network_chaos.yml#L89